scriptmodules/emulators/ags.sh

```diff
@@ -11,45 +11,79 @@
 
 rp_module_id="ags"
 rp_module_desc="Adventure Game Studio - Adventure game engine"
-rp_module_help="ROM Extension: .exe\n\nCopy your Adventure Game Studio roms to $romdir/ags"
+rp_module_help="ROM Extension: .exe\n\nCopy your Adventure Game Studio roms to $romdir/ags/<game>/"
 rp_module_licence="OTHER https://raw.githubusercontent.com/adventuregamestudio/ags/master/License.txt"
-rp_module_repo="git https://github.com/adventuregamestudio/ags.git release-3.5.1"
+rp_module_repo="git https://github.com/adventuregamestudio/ags.git master"
```

Comment:
Assuming they will be doing a branch / tag for the release we should use that here (When it's done).

Reply:
Yes. Right. Currently there is a lot of activity at the development/main branch.